tests/test_TSV_identification.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
```

Comment:
You now have two test files. And I noticed belatedly that the one test file used to be called test_manifest_io.py.

In Java and JavaScript, the test files' names are based on the file they are testing. That way if you in a source file, you can easily locate the corresponding tests. That sort of seems to be the pattern in Python, although I couldn't find it written down anywhere. I did find a couple of repos that seem to follow the same pattern.

I would have one test file, `test_utils.py`, and then have the two test classes inside.